hermes-upload.ps1

```diff
@@ -222,38 +222,109 @@ function ZipFolder(
         + $directory.FullName + ").")
 }
 
-function FTP-Upload
+################################################################################
+# Upload source: http://blog.majcica.com/2016/01/13/powershell-tips-and-tricks-multipartform-data-requests/
+################################################################################
+function Invoke-MultipartFormDataUpload
 {
-    param(
-        [string] $user = $null,
-        [string] $pass = $null,
-        [string] $file = $null,
-        [string] $server = $null,
-        [string] $zipFilename = $null
+    [CmdletBinding()]
+    PARAM
+    (
+        [string][parameter(Mandatory = $true)][ValidateNotNullOrEmpty()]$UserId,
+        [string][parameter(Mandatory = $true)][ValidateNotNullOrEmpty()]$ApiKey,
+        [string][parameter(Mandatory = $true)][ValidateNotNullOrEmpty()]$InFile,
+        [string]$ContentType,
```

Comment:
Mainly because the specification for the upload portion of dasher was to have Election Date be an optional value too. It's nice to have it for filing files into logical groups, but is still an optional thing.

I should probably remove $ContentType as a parameter, it was just a carryover from the source script I started with.